views/index.erb

```diff
@@ -50,44 +45,23 @@
           <li>Toothbrush &amp; Toiletries</li>
           <li>Towel</li>
           <li>Sleeping Bag</li>
-          <li>Earplugs</li>
```

Comment:
Have snorers been banned from attending ? Or are we all getting our own rooms ?(!) Otherwise I'd really leave this on the list. No earplugs + room with snoring person = very long, distressing night :(